lib/src/presentation/scenes/auth/auth_bloc.dart

```diff
@@ -6,23 +6,32 @@ import 'package:xhabits/src/domain/validation/username_validation.dart';
 import 'package:xhabits/src/presentation/scenes/auth/auth_state.dart';
 
 class AuthBloc {
```

Comment:
@Babiy7 there should be two different bloc objects for each screen.
LoginScreenBloc and SignUpScreenBloc
All the logic should be separated